src/desktop/components/split_test/running_tests.coffee

```diff
@@ -9,7 +9,9 @@
 #   edge: 'new'
```

Comment:
update the docs for `outcomes` to specify accepted values (`array` or `object` with values)